src/app/public/modules/radio/radio-group.component.ts

```diff
@@ -87,16 +90,18 @@ export class SkyRadioGroupComponent implements AfterContentInit, ControlValueAcc
   private _tabIndex: number;
 
   public ngAfterContentInit(): void {
-    this.updateCheckedRadioFromValue();
-    this.updateRadioButtonNames();
-    this.updateRadioButtonTabIndexes();
+    this.resetRadioButtons();
 
     // Watch for radio selections.
     this.radios.forEach((radio) => {
       radio.change.subscribe((change: SkyRadioChange) => {
         this.writeValue(change.value);
       });
     });
+
+    this.radios.changes.subscribe(() => {
```

Comment:
Do we need a `.takeUntil()` on this subscribe?
I also noticed there's a leftover `.subscribe()` from before on line `97`.

Reply:
Fixed

Reply:
As far as "there's a leftover `.subscribe()` from before on line `97`" I'm not sure what you mean there. That is for different functionality. I did add the `.takeUntil()` there too if that is what you meant.